### PR TITLE
Feature/how to section

### LINKED ISSUE #15: UI: "How to shield"
From reqs: 

> “How to shield” and “What happens to shielded funds” should be obvious to first-time visitors.

Let's include a "how to shield" section with appropriate copy. My first thought is that this belongs as a top level component that's visible just beside or below the form, but open to other ideas. Design-wise, should be considered together with #16

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Add how to section under the shielding form

Closes #15 #16 

Screenshot from 2023-01-09 09-25-22

### CODE REVIEW

File: components/HowTo.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      marginBottom="3rem"
+    >
+      <Heading as="h1" size="lg" alignSelf="center">
+        Tutorial
+      </Heading>
+      <Heading as="h4" size="sm" marginTop="1rem">
+        How to shield?
+      </Heading>
+      <OrderedList marginTop="1rem">
+        <ListItem>Create a Railgun wallet</ListItem>
```
Comment (mds1) [inline components/HowTo.tsx:22]: How do you do this? Can we link to a guide / app that helps with this?

File: components/HowTo.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      padding="2rem"
+      maxWidth="40rem"
+      marginTop="3rem"
+      marginBottom="3rem"
+    >
+      <Heading as="h1" size="lg" alignSelf="center">
+        Tutorial
+      </Heading>
+      <Heading as="h4" size="sm" marginTop="1rem">
+        How to shield?
```
Comment (mds1) [inline components/HowTo.tsx:19]:
```suggestion
        How to shield funds?
```

Comment (alexkeating) [reply components/HowTo.tsx:19]: I think your understanding is correct here

File: components/HowTo.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      </Heading>
+      <Heading as="h4" size="sm" marginTop="1rem">
+        How to shield funds?
+      </Heading>
+      <OrderedList marginTop="1rem">
+        <ListItem>
+          Create a Railgun wallet on{" "}
+          <Link href="https://app.railway.xyz/" isExternal>
+            Railway
+          </Link>
```
Comment (mds1) [inline components/HowTo.tsx:33]: One nit is the links currently have no style until you hover, so you can't tell it's a link. Up to you if you want to fix in this PR or track in an issue

image

Comment (alexkeating) [reply components/HowTo.tsx:33]: Yes, good call, I will create a ticket and we can handle styling towards the end. Does that sound good @wildmolasses

Comment (alexkeating) [reply components/HowTo.tsx:33]: #33

**Review by mds1 — APPROVED** [review]

**Pull request merged**